site/src/pages/components/accordion.research.mdx

```diff
@@ -0,0 +1,33 @@
+---
+name: Accordion
+
+menu: Research
+layout: ../../layouts/ComponentLayout.astro
+---
+
+An accordion is a sequence of
+[disclosure](/components/disclosure.research)
+widgets in a row.
+
+Some accordions are exclusive,
+which means that at any moment in time,
+only zero or one of the disclosure widgets can be open.
+In an exclusive accordion, opening one of the items
+closes any existing open item.
+(Of the design systems below,
+the accordion for Semantic UI is exclusive,
+the accordion for Stardust UI can be exclusive or not exclusive,
+and those for Carbon Design System, FAST, KoliBri, Lightning Design System,
+and Lion are not exclusive.)
```

Comment:
Does "not" mean "never" for the last list? If so, maybe s/not/never ?

Reply:
Done, and also added an "always" to the first list to match.